src/utils/get-device-size.js

```diff
@@ -0,0 +1,12 @@
+import { readSync } from 'read-file-relative';
+import find from 'array-find';
+
+
+const DEVICE_DATABASE = JSON.parse(readSync('../../data/devices.json'));
+
+
+export default function (deviceName) {
+    deviceName = deviceName.toLowerCase().split(' ').join('');
+
+    return find(DEVICE_DATABASE, device => device.name.indexOf(deviceName) !== -1);
```

Comment:
Just thought that we can use dictionary for `DEVICE_DATABASE` so we will have O(1) lookup